src/guacd/daemon.c

```diff
@@ -304,8 +304,23 @@ int main(int argc, char* argv[]) {
 
     }
 
+    /* create socket for AF_INET6 to allow IPv6 bind in case there is no IPv4 address available */
+    sa_family_t family = AF_INET6;
+    current_address = addresses;
+    while (current_address != NULL) {
+
+        if (current_address->ai_family == AF_INET)
+        {
+            family = AF_INET;
+        }
```

Comment:
Please follow the same style as established elsewhere (opening brace on same line). See surrounding code and: http://guacamole.apache.org/guac-style/

Reply:
Sorry for missing this, fixed.